common/src/main/java/io/homeassistant/companion/android/common/sensors/DisplaySensorManager.kt

```diff
@@ -98,4 +159,110 @@ class DisplaySensorManager : SensorManager {
             mapOf()
         )
     }
+
+    private fun updateScreenOrientation(context: Context) {
+        if (!isEnabled(context, screenOrientation)) {
+            return
+        }
+
+        @Suppress("DEPRECATION")
+        val orientation = when (context.resources.configuration.orientation) {
+            Configuration.ORIENTATION_PORTRAIT -> "portrait"
+            Configuration.ORIENTATION_LANDSCAPE -> "landscape"
+            Configuration.ORIENTATION_SQUARE -> "square"
+            Configuration.ORIENTATION_UNDEFINED -> STATE_UNKNOWN
+            else -> STATE_UNKNOWN
+        }
+
+        val icon = when (orientation) {
+            "portrait" -> "mdi:phone-rotate-portrait"

```

Comment:
This is the icon to rotate from landscape to portrait, not sure about the fit 🤔

Reply:
yes the icons were not easy to pick, originally I had `crop-portrait` but this looked more fun lol

Reply:
When quickly looking at it the icons seem identical but I recognize how hard it is. I was thinking about `tablet` for landscape, `cellphone` for portrait for a while but the screen aspect ratios don't match unfortunately.

Reply:
yea this one wasnt easy based on the current available options lol

Reply:
Maybe this could be good to leave as is in case another contributor has a better idea? Picking icons and naming things are usually the most thought driven part of PRs like this :)